pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -379,6 +379,11 @@ func (c *Reconciler) resolvePipelineState(
 			}
 			return nil, controller.NewPermanentError(err)
 		}
+		if verificationResult != nil && verificationResult.VerificationResultType == trustedresources.VerificationError {
```

Comment:
It looks like this verification result is associated with a Task. this error message may be misleading.

Reply:
Good point! Thx!